web/src/pages/index.tsx

```diff
@@ -1,6 +1,7 @@
 /** @jsx */
 import { Box, Grid, Heading, Label } from '@theme-ui/components';
 import React from 'react';
+import BjerkVideo from '../assets/Bjerk.mp4';
```

Comment:
This doesn't play on my computer (MacOSX, Chrome). However, isn't it better to use WEBM?

Reply:
@cobraz Yes, I changed to the .webm